both/methods.js

```diff
@@ -649,9 +609,9 @@ if(Meteor.isServer) {
           log.l1("adding data for round " + nthRound);
           var roundId = Rounds.insert({
             nthRound: nthRound + 1,
+            totalRounds: wcaEvent.rounds.length,
```

Comment:
Nice! Glad to see you remembered to do this =). Please ignore https://github.com/cubing/ccm/pull/241/files#r31365040